### PR TITLE
Translate "Testing Overview" to German

### PR DESCRIPTION
**Pull request by mhienle**

### CODE REVIEW

File: content/docs/testing.md [hunk 1]
```diff
@@ -1,40 +1,39 @@
 ---
 id: testing
-title: Testing Overview
+title: Testing-Überblick
```
Comment (ph1p) [inline content/docs/testing.md]:
```suggestion
Test-Übersicht
```

File: content/docs/testing-recipes.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 id: testing-recipes
-title: Testing Recipes
+title: Testing-Rezepte
```
Comment (ph1p) [inline content/docs/testing-recipes.md]:
```suggestion
title: Test-Rezepte
```

File: content/docs/testing-environments.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 id: testing-environments
-title: Testing Environments
+title: Testing-Umgebungen
```
Comment (ph1p) [inline content/docs/testing-environments.md]:
```suggestion
title: Test-Umgebungen
```

File: content/docs/testing.md [hunk 4]
```diff
@@ -1,40 +1,39 @@
...
 
-**[Jest](https://facebook.github.io/jest/)** is a JavaScript test runner that lets you access the DOM via [`jsdom`](/docs/testing-environments.html#mocking-a-rendering-surface). While jsdom is only an approximation of how the browser works, it is often good enough for testing React components. Jest provides a great iteration speed combined with powerful features like mocking [modules](/docs/testing-environments.html#mocking-modules) and [timers](/docs/testing-environments.html#mocking-timers) so you can have more control over how the code executes.
+**[React Testing Library](https://testing-library.com/react)** ist eine Menge von Helpern, die dich React-Komponenten testen lassen, ohne deren Implementierungsdetails zu kennen. Dieser Ansatz macht Refactoring einfach und leitet dich außerdem zu den Best Practices für Accessibility. Obwohl sie keine Möglichkeit bietet, eine Komponente "shallow" (d.h. ohne ihre Kinder) zu rendern, lässt ein Test-Runner wie Jest das via [Mocking](/docs/testing-recipes.html#mocking-modules) zu.
```
Comment (ph1p) [inline content/docs/testing.md]:
```suggestion
**[React Testing Library](https://testing-library.com/react)** ist eine Menge von Helpern, die dich React-Komponenten testen lassen, ohne deren Implementierungsdetails zu kennen. Dieser Ansatz macht Refactoring einfach und spornt dich außerdem zu Best Practices für Barrierefreiheit an. Obwohl sie keine Möglichkeit bietet, eine Komponente "shallow" (d.h. ohne ihre Kinder) zu rendern, lässt ein Test-Runner wie Jest das via [Mocking](/docs/testing-recipes.html#mocking-modules) zu.
```

**Comment by mhienle** [comment]
Thanks for the feedback @ph1p, I committed your suggestions.

**Comment by ph1p** [comment]
> Thanks for the feedback @ph1p, I committed your suggestions.

Top! Please resolve the failed build and I'll merge it.

**Comment by mhienle** [comment]
@ph1p Fixed.

**Pull request merged**